src/components/csvLocalizer/jsonLocalizer.js

```diff
@@ -30,19 +39,35 @@ const JsonLocalizer = () => {
   }, [chosenFiles]);
 
   React.useEffect(() => {
-    console.log("DATA ", data);
+    console.log('DATA ', data);
   }, [data]);
 
+  const cellOnClick = React.useCallback((id) => {
+    setShownInput(id);
+    inputRef?.current?.focus();
+  }, []);
+
+  const onCancel = React.useCallback(() => {
+    setShownInput('');
+    setNewtranslation('');
+  }, []);
+
+  const saveTranslation = React.useCallback(() => {
+    console.log(`id: ${shownInput}`, `translation: ${newTranslation}`);
```

Comment:
Is it temporary?

Reply:
Yes, just to see we have everything that we need.